hi-ml/testhiml/testhiml/test_data_augmentations.py

```diff
@@ -0,0 +1,72 @@
+import torch
+import numpy as np
+import random
+
+from typing import Callable
+
+from health_ml.utils.data_augmentations import HEDJitter, StainNormalization
+
+
+# global dummy image
+dummy_img = torch.Tensor(
+        [[[[0.4767, 0.0415],
+          [0.8325, 0.8420]],
+         [[0.9859, 0.9119],
+          [0.8717, 0.9098]],
+         [[0.1592, 0.7216],
+          [0.8305, 0.1127]]]])
+
+def _test_data_augmentation(data_augmentation: Callable[[torch.Tensor], torch.Tensor],
+                            input_img: torch.Tensor,
+                            expected_output_img: torch.Tensor,
+                            stochastic: bool) -> None:
+    if stochastic:
+        torch.manual_seed(0)
+        np.random.seed(0)
+        random.seed(0)
```

Comment:
lightning has a "seed_everythnig" function

Reply:
Will save this for a future PR